docs/release_notes/version_chart/version_chart.md

```diff
@@ -15,13 +15,14 @@ Check to make sure the Spark version and Scala version of your Prophecy Scala Li
 of your cluster!
 :::
 
-:::info
-Query this table using the following API and your ([personal access token](..%2Fmetadata%2FprophecyAPI.md)):
+You can query this table using the following API and your ([personal access token](/docs/metadata/prophecyAPI.md)):
```

Comment:
curious why personal access token is in parentheses?

Reply:
good catch